addon/components/x-toggle/template.hbs

```diff
@@ -1,19 +1,26 @@
-{{#if showLabels}}
-	<span class="toggle-text toggle-prefix {{size}}">
-		{{offLabelValue}}
-	</span>
-{{/if}}
+<span class="x-toggle-component">
+  {{#if showLabels}}
+  	<span class="toggle-text toggle-prefix off-label" onclick={{action 'setToValue' false}}>
+  		{{_offLabel}}
+  	</span>
+  {{/if}}
+  <span
+    id={{elementId}}
+    class="x-toggle-container {{size}}{{if toggled ' x-toggle-container-checked'}}{{if disabled ' x-toggle-container-disabled'}}"
+    onclick={{action 'onClick'}}
+  >
+    {{input id=forId type='checkbox' checked=toggled class='x-toggle' disabled=disabled name=name}}
```

Comment:
I think some of the tests might be failing because they're clicking the input element. Maybe we could we set the action on the input like this
```
 <input 
    type="checkbox" 
    id={{forId}} 
    checked={{toggled}} 
    class="x-toggle" 
    disabled={{disabled} 
    name={{name}} 
    onchange={{action "setToValue value="target.checked"}}>
```